src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -299,6 +298,15 @@ public void deleteUsersByUserId(String userId, String lang,
       DeleteUsersByUserIdResponse.class, asyncResultHandler);
   }
 
+  @Validate
+  @Override
+  public void deleteUsers(String query, RoutingContext routingContext,
+          Map<String, String> okapiHeaders,
+          Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    PgUtil.delete(getTableName(null), query, okapiHeaders, vertxContext,
```

Comment:
Pesonally, I would leave a gap between a multi-line method signature and the implementation (and reduce the indent, which only takes up horizontal space)

```suggestion
    Map<String, String> okapiHeaders,
    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
    
    PgUtil.delete(getTableName(null), query, okapiHeaders, vertxContext,
```

Reply:
The new code mimics the other methods in this file for consistency.

Changing the formatting of the whole file is out of scope of this pull request, please create an issue against mod-users for this code cleanup task.

Reply:
I didn't suggest reformatting the entire file. I suggested making one simple change to improve the readability of code that was already being changed in this pull request.

Reply:
The pull request uses 8 space indentation to comply with https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html#248 suggested by https://dev.folio.org/guidelines/contributing/#style-conventions . 8 space indentation is consistently used throughout UsersAPI.java.

You suggest to switch to 4 space indentation suggested by https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent . This code style change should be made throughout UsersAPI.java for consistency, not only for a single method.

Reply:
> 8 space indentation is consistently used throughout UsersAPI.java.

That is unfortunate, as [FOLIO decided](https://dev.folio.org/guidelines/contributing/) (quoted below) long ago to diverge from the Oracle standard and use 2 spaces rather than 4 per indentation.

>  _We indent with two spaces only, because vert.x uses deeply nested callbacks._

Reply:
I care more about the vertical whitespace change. Do you also object to that?

Reply:
Yes, because the body is a single statement that is a method call. This is easy to grasp, in such cases I never add a blank line:

```
  public void deleteUsers(String query, RoutingContext routingContext,
      Map<String, String> okapiHeaders,
      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
    PgUtil.delete(getTableName(null), query, okapiHeaders, vertxContext,
         DeleteUsersResponse.class, asyncResultHandler);
  }
```

I don't see an improvement when inserting the blank line:

```
  public void deleteUsers(String query, RoutingContext routingContext,
      Map<String, String> okapiHeaders,
      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {

    PgUtil.delete(getTableName(null), query, okapiHeaders, vertxContext,
         DeleteUsersResponse.class, asyncResultHandler);
  }
```

However, if the body consists of multiple statements or a construct like a loop or if-then-else then I add a blank line because it is more likely that the line-wrapped method parameters get misread as a wrapped statement.

Reply:
> Yes, because the body is a single statement that is a method call. This is easy to grasp, in such cases I never add a blank line:

I find it difficult to differentiate between the multi-line method signature (which has indentation) and the body of the method. To me, this also applies when the body is only a single statement (especially when that statement is split across multiple lines).

I'll admit, your answer surprises me, as I think I recall a conversation we had where you asked for whitespace between (dependency maybe) entries in a pom file in order to help distinguish between them.

Anyway, I'll take this comment as another rejection of my feedback.